packages/hoppscotch-common/src/helpers/import-export/import/postmanEnv.ts

```diff
@@ -17,30 +16,40 @@ const postmanEnvSchema = z.object({
   ),
 })
 
-export const postmanEnvImporter = (content: string) => {
-  const parsedContent = safeParseJSON(content)
+type PostmanEnv = z.infer<typeof postmanEnvSchema>
 
-  // parse json from the environments string
-  if (O.isNone(parsedContent)) {
+export const postmanEnvImporter = (contents: string[]) => {
+  const parsedContents = contents.map((str) => safeParseJSON(str, true))
+  if (parsedContents.some((parsed) => O.isNone(parsed))) {
     return TE.left(IMPORTER_INVALID_FILE_FORMAT)
   }
 
-  const validationResult = postmanEnvSchema.safeParse(parsedContent.value)
+  const parsedValues = parsedContents.flatMap((parsed) => {
+    const unwrappedEntry = O.toNullable(parsed) as PostmanEnv[] | null
 
-  if (!validationResult.success) {
-    return TE.left(IMPORTER_INVALID_FILE_FORMAT)
-  }
+    if (unwrappedEntry) {
+      return unwrappedEntry.map((entry) => ({
+        ...entry,
+        values: entry.values?.map((valueEntry) => ({
+          ...valueEntry,
+          value: String(valueEntry.value),
```

Comment:
As mentioned above, we use the safeguard to prevent exceptions if attempting to import files from other platforms. Also, the environment variable values are stringified as expected by the system.